diesel/src/query_builder/insert_statement.rs

```diff
@@ -351,3 +351,66 @@ impl<'a, T, Table> UndecoratedInsertRecord<Table> for &'a Vec<T> where
     &'a [T]: UndecoratedInsertRecord<Table>,
 {
 }
+
+/// The structure returned by [`insert_default_values`](/diesel/fn.insert_default_values.html). The
+/// only thing that can be done with it is call `into`.
+#[derive(Debug, Copy, Clone, Default)]
+pub struct IncompleteDefaultInsertStatement {}
+
+#[derive(Debug, Copy, Clone)]
+pub struct DefaultInsertStatement<T, Ret=NoReturningClause> {
+    target: T,
+    returning: Ret,
+}
+
+impl IncompleteDefaultInsertStatement {
+    #[doc(hidden)]
+    pub fn new() -> Self {
+        IncompleteDefaultInsertStatement {}
+    }
+
+    /// Specify which table the data passed to `insert` should be added to.
```

Comment:
Not in this PR, but maybe in the future we should add examples to these incomplete methods to show them used in context. Or at least link to where they are used in context.